content/07.pathogenesis.md

```diff
@@ -324,9 +324,9 @@ Viral particles can enter the body if they then come in contact with the oral, n
 Droplet transmission occurs when a contagious individual sneezes, coughs, or exhales and produces respiratory droplets that can contain a large number of viral particles [@pubmed:23771256].
 Contact with these droplets can occur either through direct exposure to the droplets, such as breathing in droplets produced by a sneeze, or exposure to particles that have settled on a surface [@pubmed:23771256].
 Aerosol transmission refers to much smaller particles (less than 5 micrometers) that are also produced by sneezing, coughing, or exhaling [@doi:10.1097/QCO.0000000000000563; @pubmed:23771256].
-The small size of these particles allows them to remain suspended over a longer period of time and potentially to be moved by air currents [@pubmed:23771256].
+The small size of these particles allow them to remain suspended over a longer period of time and potentially to be moved by air currents [@pubmed:23771256].
```

Comment:
"size... allows" reads better to me here

Reply:
```suggestion
The small size of these particles allows them to remain suspended over a longer period of time and potentially to be moved by air currents [@pubmed:23771256].
```
I agree with @agitter, it simplifies to "The size allows them to..."